updater/osv/osv.go

```diff
@@ -594,12 +598,13 @@ func (e *ecs) Insert(ctx context.Context, name string, a *advisory) (err error)
 				vs = b.String()
 			}
 			pkgName := af.Package.PURL
-			if af.Package.Ecosystem == ecosystemMaven {
+			if af.Package.Ecosystem == ecosystemMaven || af.Package.Ecosystem == ecosystemPyPI {
```

Comment:
nit: maybe a switch here, too? might be more readable and extensible than the if in this context